components/blocks/youTube.js

```diff
@@ -2,41 +2,33 @@ import React from "react";
 
 import styles from "./youtube.module.css";
 
-const YouTube = ({ caption, videoId, start, end }) => {
+const YouTube = ({ caption, videoId, start, end, autoplay, loop }) => {
   let YouTubeBlock;
+  let mute = 0;
 
-  if (caption) {
-    YouTubeBlock = (
-      <section className={styles.Container}>
-        <section className={styles.IframeContainer}>
-          <iframe
-            src={`https://www.youtube-nocookie.com/embed/${videoId}?rel=0&start=${start}&end=${end}`}
-            className={styles.Iframe}
-            title="YouTube video player"
-            frameBorder="0"
-            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-            allowFullScreen
-          ></iframe>
-        </section>
-        <section className={styles.CaptionContainer}>
-          <p className={styles.Caption}>{caption}</p>
-        </section>
-      </section>
-    );
-  } else {
-    YouTubeBlock = (
+  if (autoplay || loop) {
+    mute = 1;
+  }
+
+  YouTubeBlock = (
+    <section className={styles.Container}>
       <section className={styles.IframeContainer}>
```

Comment:
Can we change this to a `<div>` component, for semantic's purpose?

Reply:
Good call! Fixed on 331de0 I hope

Reply:
I was actually referring to the inner container, so more like this:

```
<section className={styles.Container}>
  <div className={styles.IframeContainer}>
    ...
  </div>
</section>
```

Sorry if it wasn't clear enough!

Reply:
😬 My bad! Creating a new PR now